conf/application.conf

```diff
@@ -88,6 +88,19 @@ microservice {
       host = localhost
       port = 9184
     }
+
+    hip {
+      stub-url = "http://localhost:9156"
```

Comment:
(see previous comment) Could we have hip "host"  and "port", and use the stub in the appropriate env config files?